bin/go-bindata.sh

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env bash
 
 WD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-ROOT=$(dirname $WD)
+ROOT="$(dirname "$WD")"
 
 # Ensure expected GOPATH setup
-if [ $ROOT != "${GOPATH-$HOME/go}/src/istio.io/istio" ]; then
+if [ "$ROOT" != "${GOPATH-$HOME/go}/src/istio.io/istio" ]; then
   die "Istio not found in GOPATH/src/istio.io/"
 fi
 
 gen_img=gcr.io/istio-testing/go_generate_dependency:2018-07-26
 
-docker run  -i  --rm --entrypoint go-bindata -v $ROOT:$ROOT -w $(pwd) $gen_img $*
+# shellcheck disable=SC2048
+docker run  -i  --rm --entrypoint go-bindata -v "$ROOT":"$ROOT" -w "$(pwd)" "$gen_img" $*
```

Comment:
Style nit: you might prefer quoting the whole expression: `"$ROOT:$ROOT"`, but up to you.
`"$gen_img"` does not _need_ to be quoted. Also up to your style preferences.

Reply:
can we add a newline here as well?

Reply:
Thanks Doug for review! But am closing this pr as @Tahler is taking over this stuff